[Regression] CLI: filter_info missing @classmethod

```
self = <tests.foreman.cli.test_contentviewfilter.ContentViewFilterTestCase testMethod=test_positive_create_by_org_label>

    @tier1
    def test_positive_create_by_org_label(self):
        """Create new content view filter and assign it to existing content
            view by name. Use organization label for reference
    
            @id: f233e223-c08c-4ce1-b87a-9e055fdd7b83
    
            @Assert: Content view filter created successfully
    
            """
        cvf_name = gen_string('utf8')
        ContentView.filter_create({
            'content-view': self.content_view['name'],
            'inclusion': 'true',
            'name': cvf_name,
            'organization-label': self.org['label'],
            'type': 'erratum',
        })
        ContentView.filter_info({
            u'content-view-id': self.content_view['id'],
>           u'name': cvf_name,
        })
E       TypeError: unbound method filter_info() must be called with ContentView instance as first argument (got dict instance instead)

tests/foreman/cli/test_contentviewfilter.py:234: TypeError
```

This used not to  be a problem, looks like something changed in the robottelo core function and it now requires an instance to be executed on.

got it, `@classmethod` is missing

--- a/robottelo/cli/contentview.py
+++ b/robottelo/cli/contentview.py
@@ -164,6 +164,7 @@ class ContentView(Base):
         return cls.execute(
             cls._construct_command(options), output_format='csv')
 
+    @classmethod
     def filter_info(cls, options):
         """Provides filter info related to content-view's version."""
 

